cmd/aem/gts.go

```diff
@@ -0,0 +1,197 @@
+		Aliases: []string{},
+		Short:   "Create Global Trust Store",
+		Run: func(cmd *cobra.Command, args []string) {
+			instance, err := c.aem.InstanceManager().One()
+			if err != nil {
+				c.Error(err)
+				return
+			}
+
+			trustStorePassword, _ := cmd.Flags().GetString("password")
+
+			changed, err := instance.GlobalTrustStore().Create(trustStorePassword)
+
+			if err != nil {
+				c.Error(err)
+				return
+			}
+
+			if changed {
+				c.Changed("Global trust store created")
+			} else {
+				c.Ok("Global trust store already exists")
+			}
+		},
+	}
+
+	cmd.Flags().String("password", "", "Password to Global Trust Store")
+	_ = cmd.MarkFlagRequired("password")
+
+	return cmd
+}
+
+func (c *CLI) gtsCertCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "certificate",
+		Short: "Manage Global Trust Store Certificates",
+	}
+	cmd.AddCommand(c.gtsCertAddCmd())
+	cmd.AddCommand(c.gtsCertRemoveCmd())
+	cmd.AddCommand(c.gtsCertReadCmd())
+	return cmd
+
+}
+
+func (c *CLI) gtsCertAddCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "add",
+		Aliases: []string{},
+		Short:   "Add Certificate to Global Trust Store",
+		Run: func(cmd *cobra.Command, args []string) {
+			instance, err := c.aem.InstanceManager().One()
+			if err != nil {
+				c.Error(err)
+				return
+			}
+
+			certificateFilePath, _ := cmd.Flags().GetString("path")
+
+			certificate, changed, err := instance.GlobalTrustStore().AddCertificate(certificateFilePath)
+
+			if err != nil {
+				c.Error(err)
+				return
+			}
+
+			c.SetOutput("Certificate", certificate.Alias)
+
+			if changed {
+				c.Changed("Certificate added")
+			} else {
+				c.Ok("Certificate already exists")
+			}
+		},
+	}
+
+	cmd.Flags().String("path", "", "Certificate file path (PEM|DER format)")
+	_ = cmd.MarkFlagRequired("path")
+
+	return cmd
+}
+
+func (c *CLI) gtsCertRemoveCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "remove",
+		Aliases: []string{},
+		Short:   "Remove Certificate from Global Trust Store",
+		Run: func(cmd *cobra.Command, args []string) {
+			instance, err := c.aem.InstanceManager().One()
+			if err != nil {
+				c.Error(err)
+				return
+			}
+
+			certifiacteAlias, _ := cmd.Flags().GetString("alias")
+			changed, err := instance.GlobalTrustStore().RemoveCertificate(certifiacteAlias)
+
+			if err != nil {
+				c.Error(err)
+				return
+			}
+
+			if changed {
+				c.Changed("Certificate removed")
+			} else {
+				c.Ok("Certificate not found")
+			}
+		},
+	}
+
+	cmd.Flags().String("alias", "", "Certificate Alias")
+	_ = cmd.MarkFlagRequired("alias")
+
+	return cmd
+}
+
+func (c *CLI) gtsCertReadCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "read",
+		Aliases: []string{},
+		Short:   "Read certificate from Global Trust Store",
+		Run: func(cmd *cobra.Command, args []string) {
+			instance, err := c.aem.InstanceManager().One()
+			if err != nil {
+				c.Error(err)
+				return
+			}
+
+			certifiacteAlias, _ := cmd.Flags().GetString("alias")
```

Comment:
you are in the scope of certificate (method name has certificate keyword)  so your variable could be just `alias`